### ISSUE TITLE
is it possible to get old beta running in browser 

### ISSUE THREAD

**Issue by broiscool** [LAX1DUDE/eaglercraft#16]
if their is a way to play it please tell me

**Comment by broiscool** [comment]
because i want to play old beta at shool

**Comment by LAX1DUDE** [comment]
Yeah I can figure that out at some point. I am currently making a mod loader but old beta is a pretty simple thing to get running in this environment. All emulation required to run old beta is already implemented in the net.lax1dude.eaglercraft package.

**Comment by broiscool** [comment]
but is thare all redey a way to play beta in browser

**Comment by LAX1DUDE** [comment]
no, I said I’ll figure it out
